runtime/gc_stats/ScavengerJavaStats.cpp

```diff
@@ -65,4 +71,8 @@ MM_ScavengerJavaStats::mergeOwnableSynchronizerCounts(MM_ScavengerJavaStats *sta
 	_ownableSynchronizerCandidates += statsToMerge->_ownableSynchronizerCandidates;
 	_ownableSynchronizerTotalSurvived += statsToMerge->_ownableSynchronizerTotalSurvived;
 	_ownableSynchronizerNurserySurvived += statsToMerge->_ownableSynchronizerNurserySurvived;
+
+	// _monitorReferenceCleared += statsToMerge->_monitorReferenceCleared;
+	// _monitorReferenceCandidates += statsToMerge->_monitorReferenceCandidates;
```

Comment:
@RSalman Should I delete these two lines? `type="scavenge"` works without them. I believe they are merged in here:
https://github.com/eclipse/openj9/blob/ad023b2e405300e319bff32e728bd1cfef2defab/runtime/gc_glue_java/ScavengerDelegate.cpp#L227
Would like to confirm with you.

Reply:
Yes, these lines shouldn't be here